jep-011-let-function.md

```diff
@@ -4,8 +4,8 @@
 |---|---
 | **JEP**    | 11
 | **Author** | James Sayerwinnie
-| **Status** | draft
 | **Created**| 24-Feb-2015
+| **SemVer** | MINOR
```

Comment:
See [this discussion](https://github.com/jmespath-community/jmespath.jep/pull/30/files#diff-45659dae3c8acfb7058b31c52c8600ca9c72a11a482372feea9ab85cb0f270faR9) before approval.